docs/guide/architecture-styles/event-driven-content.md

```diff
@@ -42,7 +42,7 @@ There are two primary topologies within many event-driven architectures:
 - Producers and consumers are decoupled.
```

Comment:
I'm thinking to mention the evolutionary feature of EDA in here. For example,

> No point-to-point integrations are required, making it easy to add new consumers to the system. This allows for easier evolution of the system.

What's your advice?

Reply:
Being "evolutionary" is encouraged in another page, so I would suggest mentioning it here. https://learn.microsoft.com/en-us/azure/architecture/guide/design-principles/design-for-evolution

Reply:
You can add that, but please avoid terms like "easy" - that's not for you to decide and is so non specific, it sounds like marketing junk.

Just state what & how it supports workload evolution over time. Let's do that in a different PR. I need to move forward today and don't want this one blocked